api/cloudcontroller/ccv2/space_summary_test.go

```diff
@@ -0,0 +1,127 @@
+package ccv2_test
+
+import (
+	"net/http"
+
+	"code.cloudfoundry.org/cli/api/cloudcontroller/ccerror"
+	. "code.cloudfoundry.org/cli/api/cloudcontroller/ccv2"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/ghttp"
+)
+
+var _ = Describe("SpaceSummary", func() {
+	var client *Client
+
+	BeforeEach(func() {
+		client = NewTestClient()
+	})
+
+	Describe("GetSpaceSummary", func() {
+		When("there are no errors", func() {
+			BeforeEach(func() {
+				response := `{
+				 "name": "space-name",
+				 "apps": [
+						{
+							 "service_names": [
+									"service-instance-name"
+							 ],
+							 "name": "app-name"
+						}
+				 ],
+				 "services": [
+						{
+							 "name": "service-instance-name",
+							 "last_operation": {
+									"type": "create",
+									"state": "succeeded",
+									"description": "a description",
+									"updated_at": "some time",
+									"created_at": "some time"
+							 },
+							 "shared_from": null,
+							 "shared_to": [],
+							 "service_plan": {
+									"guid": "plan-guid",
+									"name": "simple-plan",
+									"service": {
+										 "guid": "service-guid",
+										 "label": "service-label"
+									}
+							 }
+						}
+				 ]
+			}`
+				server.AppendHandlers(
+					CombineHandlers(
+						VerifyRequest(http.MethodGet, "/v2/spaces/some-guid/summary"),
+						RespondWith(http.StatusOK, response, http.Header{"X-Cf-Warnings": {"warning-1"}}),
+					))
+			})
+
+			It("returns the result and all warnings", func() {
+				spaceSummary, _, err := client.GetSpaceSummary("some-guid")
```

Comment:
Please extract the invocation of the method under test into a JustBeforeEach block at the top level of the `Describe("GetSpaceSummary")...` block and use a name other than `err` for the variable that stores the returned error (to avoid shadowing).

Reply:
Done.